service/rpc/versions.go

```diff
@@ -24,6 +24,6 @@ const (
 )
 
 var (
-	MiddlewareVersion = "1.0.0"
+	MiddlewareVersion = "1.0.1"
```

Comment:
From the README it seems like Rosetta has recently moved to full semantic versioning with very strict guidelines for what entails a breaking change. By the README definition, it seems like almost anything would count as breaking? Though in this case, I went with a patch upgrade since this is fixing a clear bug preventing the expected response from coming through. (It still may count as a breaking change by the standards in the README since technically it would now return a valid response for a request that previously crashed the node, but idk 🤷‍♀️). Lmk what you think @palango

Reply:
I agree with the patch version upgrade, as it's something broken that now works.